google-home-community.groovy

```diff
@@ -562,6 +578,112 @@ private deviceTraitPreferences_Dock(deviceTrait) {
     }
 }
 
+@SuppressWarnings(['UnusedPrivateMethod', 'MethodSize'])
+private deviceTraitPreferences_EnergyStorage(deviceTrait) {
+    googleEnergyStorageDistanceUnitForUX = [
+        "KILOMETERS":      "Kilometers",
+        "MILES":           "Miles",
+    ]
+    googleCapacityUnits = [
+        "SECONDS":         "Seconds",
+        "MILES":           "Miles",
+        "KILOMETERS":      "Kilometers",
+        "PERCENTAGE":      "Percentage",
+        "KILOWATT_HOURS":  "Kilowatt Hours",
+    ]
+    section("Energy Storage Settings") {
+        input(
+            name: "${deviceTrait.name}.isRechargeable",
+            title: "Rechargeable",
+            type: "bool",
+            defaultValue: false,
+            required: true,
+            submitOnChange: true
+        )
+        input(
+            name: "${deviceTrait.name}.queryOnlyEnergyStorage",
+            title: "Query Only Energy Storage",
+            type: "bool",
+            defaultValue: false,
+            required: true,
+            submitOnChange: true
+        )
+        if (!deviceTrait.queryOnlyEnergyStorage) {
+            input(
+                name: "${deviceTrait.name}.chargeCommand",
+                title: "Charge Command",
+                type: "text",
+                required: true
+            )
+        }
+        input(
+            name: "${deviceTrait.name}.capacityRemainingRawValue",
+            title: "Capacity Remaining Value",
+            type: "text",
+            defaultValue: "battery",
+            required: true
+        )
+        input(
+            name: "${deviceTrait.name}.capacityRemainingUnit",
+            title: "Capacity Remaining Unit",
+            type: "enum",
+            options: googleCapacityUnits,
+            multiple: false,
+            required: true,
+            submitOnChange: true
+        )
```

Comment:
This needs a `defaultValue` to match what's stated in the README.

Reply:
Added and corrected the README to match "PERCENTAGE".